apps/desktop/src-tauri/config.json

```diff
@@ -1,4 +1 @@
-{
-    "skip-indexing": true,
-    "github_client_id": "5620006101aa9914ab43"
-}
+{}
```

Comment:
can we make this config file optional?

Reply:
It's not straightforward to optionally bundle additional files in Tauri (https://tauri.app/v1/guides/building/resources). Will stick this on the backburner.